frontend/pages/play/[roomcode].vue

```diff
@@ -20,14 +21,47 @@ definePageMeta({
 });
 
 const localeRoute = useLocaleRoute();
+const route = useRoute();
 
 const game = useGameStore();
+const ableToSelectExistingTeams = ref(false);
+const roomWithAllExistingTeamsAndMembers = reactive({});
 
-const { data: joinSuccess } = useAsyncData("team-join", async () => {
+const inactiveTeamsToDisplay = computed(() => {
+  return roomWithAllExistingTeamsAndMembers.value.teams.filter(
+    (team) => team.active === 0,
```

Comment:
this is more resistant to changes in the backend as both `true` and `1` are truthy and both `false` and `0` are falsy
```suggestion
    (team) => !!team.active,
```